plugins/catalog-react/api-report.md

```diff
@@ -399,6 +399,7 @@ export type EntityRefLinkProps = {
   defaultNamespace?: string;
   title?: string;
   children?: React_2.ReactNode;
+  noIcon?: boolean;
```

Comment:
Usually it's advocated to avoid negative words, [one of examples](https://www.baeldung.com/cs/conditionals#:~:text=Avoiding%20Negative%20Conditionals,-Avoiding%20negative%20conditionals&text=It%20makes%20code%20easier%20to,conditions%20rather%20than%20negative%20ones.)

Reply:
Fixed, also updated all occurrences as the original API hasn't been released in mainline yet.

Reply:
Prolly change `noTooltip` as well then? It's not really hidden so I think it might be best to go with `disableTooltip`, which MUI also uses very frequently.

Reply:
Done